diff/parser/parser.go

```diff
@@ -3,11 +3,10 @@ package parser
 import (
 	"compress/gzip"
 	"encoding/xml"
-	"os"
-	"strconv"
-
 	"github.com/omniscale/imposm3/element"
 	"github.com/omniscale/imposm3/logging"
+	"os"
+	"strconv"
```

Comment:
Can you make sure that go packages come before local packages? gofmt should sort each block individually if you put a blank line between. Or use goimports.